clang/include/clang-c/Dependencies.h

```diff
@@ -550,6 +550,12 @@ CINDEX_LINKAGE CXCStringArray
 CINDEX_LINKAGE CXCStringArray
     clang_experimental_DepGraphModule_getBuildArguments(CXDepGraphModule);
 
+/**
+ * @returns the CASID of the include-tree for this module, if any.
+ */
+CINDEX_LINKAGE const char *
```

Comment:
Maybe mention what the lifetime of the string being returned is?

Reply:
Thanks for pointing that out. I also documented that on the `getCacheKey` APIs.